moderatorFrontend/build.gradle.kts

```
@@ -1,5 +1,8 @@
 plugins {
   kotlin("js")
+
+  // Workaround for https://youtrack.jetbrains.com/issue/KT-51921 respectively https://github.com/JetBrains/kotlin-wrappers/issues/1077
+  id("com.github.turansky.kfc.legacy-union") version "4.88.0"
```

```suggestion
  id("io.github.turansky.kfc.legacy-union") version "5.8.0"
```
Suggested by Viktor [here](https://youtrack.jetbrains.com/issue/KT-51921#focus=Comments-27-5986480.0-0).